core/controllers/admin.py

```diff
@@ -70,6 +71,22 @@ def get(self):
         self.render_template('admin-page.mainpage.html')
 
 
+class AdminSessionHandler(webapp2.RequestHandler):
```

Comment:
Gosh no. Why are we doing this?

Per my discussion with @BenHenning you should not be creating new users, just have a handler that loads the dummy data using the SYSTEM_COMMITTER_ID or similar. There isn't even any dev mode guard here.

Reply:
This was just a sample pr to verify if I could log in using curl commands. I was about to update this and make it ready for review today

There are few functions that require a user_id to in publishing the topics
https://github.com/oppia/oppia/blob/develop/core/controllers/admin.py#L417

Reply:
Also in the reload exploration 
https://github.com/oppia/oppia/blob/develop/core/controllers/admin.py#L227

Reply:
Yup, I talked to @BenHenning about this and I think you can use SYSTEM_COMMITTER_ID for that, right?

Reply:
Also I think my suggestion [here](https://github.com/oppia/oppia/pull/13009#issuecomment-858198817) still stands. You'll want a full design review here before proceeding -- I have a feeling, just based on what I'm hearing in this thread, that parts of this are probably not being approached correctly.